### ISSUE TITLE
(Accessibility) Unable to Re-load “History” for Two Keyword Search Input After Clicking on the Character Added/ deleted Information

### ISSUE THREAD

**Issue by swadas** [twg/wikiwash#220]
**Issue Summary:**  (Accessibility) Unable to Re-load “History” for Two Keyword Search Input After Clicking on the Character Added/ deleted Information

**Environment:** 
Browser:  Chrome, Safari
OS: Windows 8.1, OSX El Capitan 10.11.5

**Description:** 
There is no log in the history tab as well as no Wikipedia page is found on reloading result of a two-key word input search when multiple comments are clicked.

**Steps to reproduce:** 
1.	Open a browser on Chrome or Safari. 
2.	Go to link “http://en.wikiwash.org” 
3.	Search by “Computer Science” . 
4.	Click on any revision ( do not click on any hyperlinks, such as username, address, comments). It adds 25 in between %20 with the wikiwash URL of the page. 
5.	Thus, reloading of the page fails.

**Severity:** High 
**Reproducible:**  Yes, (10/10) 

<img width="1280" alt="error loading 1" src="https://cloud.githubusercontent.com/assets/6794141/15654268/b1d8f26e-2660-11e6-877a-fd9b08d65db4.png">
<img width="1280" alt="error loading 2" src="https://cloud.githubusercontent.com/assets/6794141/15654277/c8f594e8-2660-11e6-8a7d-b5d1a10948ef.png">
